guides/image-sources/provisioning-installer-pxeboot.plantuml

```diff
@@ -0,0 +1,61 @@
+@startuml
+
+!include foreman.pstyle
+
+title Installer-based provisioning with PXE boot on bare metal
+
+actor       User
+participant "Provisioned\nHost" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
+'participant "Pulp\n(Katello)" as Pulp
+participant TFTP
+participant DHCP
+participant DNS
+!if ($puppet)
+participant "Puppet\nserver" as Puppet
+!endif
+
+note over Host : powered off
+
+== Create host in Foreman ==
+
+!include prov-create-host-pxe.iuml
+
+== Boot into OS installer ==
+
+User -> Host : powers on the machine
+Host -> DHCP : requests the reserved IP and PXE details
+Host -> TFTP : downloads bootloader
```

Comment:
I must admit that my knowledge is pretty rusty here, but I was under the impression that the machine was not downloading the bootloader again.
If I think about it, it does make sense, but still, it's surprising to learn :D

cc @ares Is it really true that we download the bootloader from TFTP every single time the machine is booted? No caching involved?